diagrams/presentation-graph.svg

```diff
@@ -11,16 +11,15 @@
 			.credential-proof > rect { fill: #bda}
 			.presentation-proof > rect { fill: #acf}
 			.claim > rect { fill: #fe9}
-		  .background { stroke: none; fill: #fff}
+		  #background { stroke: none; fill: #fff}
 
   	</style>
 		<marker id="arrow" viewBox="0 0 6 4" refX="6" refY="2" fill="#000"
       markerWidth="6" markerHeight="4" orient="auto">
     <path d="M0 0v4l6 -2z" />
     </marker>
 	</defs>
-
-	<rect class="background" width="864" height="958"/><!-- Background -->
+	<rect id="background"/><!-- Background -->
 
```

Comment:
[Unrelated to this line due to github limitiation] "AlumniOf" below should be "alumniOf".